lbz/events/api.py

```diff
@@ -87,13 +110,3 @@ def _create_eb_entry(self, new_event: BaseEvent) -> PutEventsRequestEntryTypeDef
             "Resources": self._resources,
             "Source": self._source,
         }
-
-    def _mark_sent(self) -> None:
```

Comment:
_FYI:_ After the last changes, both methods became single lines that were used in one place - did not make much sense to keep them.